pkg/users/dialog-utils.js

```diff
@@ -64,3 +64,10 @@ function show_error_dialog(title, message) {
 export function show_unexpected_error(error) {
     show_error_dialog(_("Unexpected error"), error.message || error);
 }
+
+export function is_valid_char_name(c) {
+    return (c >= 'a' && c <= 'z') ||
+        (c >= 'A' && c <= 'Z') ||
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18271-20230131-133441-a322fc08-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)